.github/ISSUE_TEMPLATE/3_feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for oneDPL
+title: ''
+labels: feature_request
+assignees: ''
+
+---
+
+**Summary**
```

Comment:
Please add a colon after your bolded headers and be consistent on capitalization: **Problem Statement:**

Reply:
Done.